.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
It's a good practice to include a trailing slash for directories in `.gitignore` to clearly indicate that it's a directory. Consider changing `.venv/` to `.venv/`.